src/routes/Books.js

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 // import { v4 as uuidv4 } from 'uuid';
```

Comment:
- [ ] Kindly make sure you remove all commented and unused code lines from your code base to make it cleaner, professional and for best practices